File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    answer = ''
    for char in s:
        count = 0
        while count < index:
            if ord(char) + 1 > 122:
                char = chr(ord(char) + 1 - 26)
            else:
                char = chr(ord(char) + 1)
            if skip.__contains__(chr(ord(char))):
                count += 0
            else:
                count += 1
        answer += char
    return answer

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.swift
import Foundation

func solution(_ s:String, _ skip:String, _ index:Int) -> String {
    var answer = ""
    let skipList = skip.map{ $0 }
    let alphabets = "abcdefghijklmnopqrstuvwxyz".filter { !skipList.contains($0) }.map { $0 }.sorted()
    for char in s {
        var i = alphabets.firstIndex(of: char)!
        i = i + index < alphabets.count ? i + index : (i + index) % alphabets.count
        answer.append(alphabets[i])
    }
    return answer
}